### ISSUE TITLE
Documentation for beginners 

### ISSUE THREAD

**Issue by sunsetjesus** [AliaksandrSiarohin/first-order-model#94]
please improve the documentation regarding training our own model. Where the checkpoints are generated? path-to-the-checkpoint is required when animating an image

**Comment by AliaksandrSiarohin** [comment]
The code will create a folder in the log directory (each run will create a time-stamped new directory).  **Checkpoints will be saved to this folder.**  To check the loss values during training see log.txt. You can also check training data reconstructions in the train-vis subfolder. By default the batch size is tunned to run on 2 or 4 Titan-X gpu (appart from speed it does not make much difference). You can change the batch size in the train_params in corresponding .yaml file.
